packages/python-packages/apiview-copilot/cli.py

```diff
@@ -1016,12 +1021,20 @@ def load_arguments(self, command):
             ac.argument(
                 "num_runs", type=int, options_list=["--num-runs", "-n"], help="Number of times to run the test case."
             )
+            ac.argument(
+                "max_parallel", 
+                type=int, 
+                options_list=["--max-parallel", "-j"], 
+                default=3,
+                help="Maximum number of evaluations to run in parallel (default: 3, set to 1 for sequential)"
+            )
```

Comment:
I would think the default should be dynamic based on the number of CPU cores you have. Might make sense to default to None and set this in code.